requirements.txt

```diff
@@ -0,0 +1,8 @@
+click==7.1.2
+Flask==1.1.2
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+pkg-resources==0.0.0
+pyserial==3.5
+Werkzeug==1.0.1
```

Comment:
Is this missing [httpie](https://github.com/michael-sbarra/voice-activated-hdmi-switch/blob/c117314e44431163af205b5c63accfad3047a43e/README.md#post-to-server-using-httpie) / not up to date from recent changes?

Reply:
httpie is optional, I added it to be cute